src/MSBuild.Bootstrap/MSBuild.Bootstrap.csproj

```diff
@@ -20,7 +20,7 @@
   <ItemGroup>
     <!-- This file is needed so the dotnet CLI knows how to map preview SDK versions to tfms (because tfms do not have preview information on them) -->
     <!-- This is because according to semver, 2.1.0-preview is not >= 2.1.0 -->
-    <Content Include="$(RepoRoot).dotnet\sdk\$(DotNetCliVersion)\Microsoft.NETCoreSdk.BundledVersions.props" CopyToOutputDirectory="PreserveNewest" />
+    <Content Include="$(NETCoreSdkBundledVersionsProps)" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
Where is this property defined? Does it point to the correct file even when building in VS or with an SDK version different from what is downloaded?

Reply:
(note this change was made a WHILE ago)



The context I remember is that `$(RepoRoot).dotnet\sdk\$(DotNetCliVersion)\Microsoft.NETCoreSdk.BundledVersions.props` was failing and `NETCoreSdkBundledVersionsProps` _looked similar_ to what  resolved to, so I went with the full path and it fixed the build issue. 

> Where is this property defined?

Here's how it's defined in https://github.com/dotnet/sdk/blob/2ec45ddf780cdb751cac4e1920d3a4e9e89dc284/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.props#L14-L16

```
  <PropertyGroup Condition="'$(NETCoreSdkBundledVersionsProps)' == ''">
    <NETCoreSdkBundledVersionsProps>$(MSBuildThisFileDirectory)..\..\..\Microsoft.NETCoreSdk.BundledVersions.props</NETCoreSdkBundledVersionsProps>
  </PropertyGroup>
```

Weird thing is that a clean (`git -xdf`) build passes with the old line of code now. It may have been something in my setup?



@dsplaisted might know more. Any reason we should or shouldn't be using `NETCoreSdkBundledVersionsProps`?

Reply:
I don't love depending on the SDK's property for this because the output will depend on the SDK used to build the project, which will change over time due to factors outside the control of the repo (since we don't lock to a version in `global.json`). We do however download a locked version (Arcade does this based on our `global.json`), so referencing that should always get the same output.